test/server/meta.test.js

```diff
@@ -1021,3 +1021,60 @@ test('should error when invalid characters are found in the subdomain - we allow
         throw new Error(`Expected error to be thrown`);
     }
 });
+
+const dataSet = [
```

Comment:
What do you think about moving this `dataSet` and test to a separate file? Perhaps we create server/meta.integration.test.js

Reply:
Sure that sounds good...